elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -438,6 +433,32 @@ pfe-navigation-item {
 
             .pfe-navigation--column:nth-child(4) {
                 -ms-grid-column: 4;
+            }               
+
+            >*:nth-child(3),
+            >*:nth-child(4) {
+                margin-top: 0;
+            }
+        }
+
+        @media (min-width: #{pfe-breakpoint(xl)}) {
+            padding-bottom: 0;
+            display: -ms-grid;
+            grid-template-columns: repeat(4, 1fr);
+            -ms-grid-columns: 1fr 1fr 1fr 1fr;
+            align-items: center;
+        }
+    }
+    &--column {
+        @media (min-width: #{pfe-breakpoint(md)}) {
+            @include browser-query(ie11) {
+                display: flex;
+                align-items: center;
+            }
+        }
+        @media (min-width: #{pfe-breakpoint(lg)}) {
+            @include browser-query(ie11) {
+                display: block;
```

Comment:
Why not display: flex; here? It would fix the vertical aligning in ie11?

Reply:
@alazzara I had it in there but it was causing all the columns to the flex display. I applied `display: flex` to IE (see line 169) and Edge (see line 279).

Reply:
@chrisdo1 Gotcha. Thanks for the info

Reply:
@changes look good. +1 for code